README.md

```diff
@@ -85,5 +85,86 @@ iex> EchoClient.echo(pid, "Hi")
 *DBG* #PID<0.379.0> received frame: {:text, "Hi"}
 ```
 
+#### start_link/3 && start/3 
+
+You can start a supervisioned WebSockex connection by using the regular **start_link/3**:
+
+```elixir
+iex> {:ok, pid} = WebSockex.start_link(url, __MODULE__, state)
+```
+
+And an unsupervisioned one (meaning it won't try to re-establish the connection on an exceptional close) with **start/3**:
```

Comment:
This isn't quite correct. It's actually only happens under one condition and that condition is when a WebSockex process is part of a supervision tree.

Reply:
So, it means the websockex process won't by default try to re-establish the connection?

Reply:
By default it will not. More than likely you were starting it from a GenServer that was part of a supervision tree. When the WebSockex exited abnormally, it caused your GenServer to exit and then the supervisor did its job and started the GenServer back up.

Just a note of caution, if your GenServer would have crashed when you used `start` then you would probably have zombie WebSockex processes. Even if you monitor them, without a link there's a chance for zombie processes. This is one of the situations where you would want to trap exits instead.

This is one of those things that is just an Erlang thing, but I will add that explanation to the README because it is relevant.